src/cases/use-cases/find-case-by-id.use-case.js

```diff
@@ -32,6 +32,16 @@ export const findCaseByIdUseCase = async (caseId) => {
 
   kase.assignedUser = userMap.get(kase.assignedUser?.id) || null;
 
+  kase.timeline = kase.timeline.map((tl) => {
+    tl.createdBy = userMap.get(tl.createdBy);
+    if (tl.data?.assignedTo) {
+      tl.data.assignedTo = userMap.get(tl.data.assignedTo);
+    }
+    tl.commentRef = mapComment(tl.comment);
+    tl.comment = undefined;
+    return tl;
+  });
+
```

Comment:
This looks like merge conflict resolve gone wrong. This code has been moved down and is already in the file